bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java

```diff
@@ -58,7 +58,7 @@ public void testSchedule() throws InterruptedException, IOException {
         long[] expectedResults = { 200, 300, 300, 300, 300 };
         for (long expectedResult : expectedResults) {
             long actualValue = times.poll().longValue();
-            assertEquals((offset + expectedResult) / 100.0, actualValue / 100.0, 0.9,
+            assertEquals((offset + expectedResult) / 100.0, actualValue / 100.0, 1.5,
```

Comment:
That makes the difference between 200 and 300 invisible. We should then probably change to 200 and 400.